### ISSUE TITLE
Transition capacitor-lab-basics into accessibility development phase

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/capacitor-lab-basics#11]
In a few weeks, @EmilyRandall will begin accessibility development and testing using capacitor-lab-basics.  We need to discuss the best way to begin that process and hand off the sim.

Initial questions that come to mind:
• Will accessibility progress exist in master or a new branch for this sim?
• Would it be beneficial for @EmilyRandall and @jessegreenberg to do some pairing on this sim during transition?
• When will this transition occur?
• What design factors need to be completed before the transition?

**Comment by jessegreenberg** [comment]
Thoughts as of 8/6/15, to be discussed again soon:

@jessegreenberg will prioritize code refactor and documentation checking in preparation for the hand off.  @EmilyRandall will work in a branch for accessibility, and will pair with @jessegreenberg to overview model and design updates.

**Comment by ariel-phet** [comment]
This issue seems no longer relevant, closing
